runtime/stdlib/contracts/crypto_test.cdc

```diff
@@ -0,0 +1,84 @@
+import Test
+
+pub var blockchain = Test.newEmulatorBlockchain()
+pub var account = blockchain.createAccount()
+
+pub fun setup() {
+    blockchain.useConfiguration(Test.Configuration({
+        "Crypto": account.address
+    }))
+
+    var crypto = Test.readFile("crypto.cdc")
+    var err = blockchain.deployContract(
+        name: "Crypto",
+        code: crypto,
+        account: account,
+        arguments: []
+    )
+
+    Test.assert(err == nil)
```

Comment:
There shouldn't really be a need to import the crypto contract, it should already be available by `import Crypto`.
If it does not work in the test framework, we should look into why and fix it.

As a result, I think the flow.json configuration file isn't needed either, and all `import Crypto from "Crypto"` can be changed to `import Crypto`

Reply:
There are two reasons that I chose not to use `import Crypto`:
1. The `I.Crypto` location is inherently excluded from code coverage (see [here](https://github.com/onflow/cadence-tools/blob/master/test/test_runner.go#L244)). This decision was made in order to avoid "noise" in integration tests. The `I.Test` location is also inherently excluded.
2. It wouldn't be much of an integration test if we used the built-in `Crypto` contract :sweat_smile: 

Having said that, is there a way to get rid of the `flow.json` config file?